src/h3lib/lib/polygon.c

```diff
@@ -57,13 +50,34 @@
  * @return              Whether the point is contained
  */
 
+/**
+ * Create a bounding box from a Geofence
+ * @name bboxFromGeofence
+ * @param geofence Input Geofence
+ * @param bbox     Output bbox
+ */
```

Comment:
Is this a case where Doxygen cannot easily determine the function name? Are there any problems with the resulting output?

Reply:
These are the functions created via macro in the include file, so there's nowhere for Doxygen to find the name (the symbol won't exist until compiled). I actually put these in to be human-readable; Doxygen doesn't do a great job of picking them up. Probably I should move them into `polygon.h`.

Reply:
Moved to `polygon.h`, which fixes the Doxygen output.

Reply:
Nice.